src/server/pps/cmds/cmds.go

```diff
@@ -759,7 +759,7 @@ you can increase the amount of memory used for the bloom filters with the
 			return client.GarbageCollect(memoryBytes)
 		}),
 	}
-	getLogs.Flags().Int64VarP(&memoryBytes, "memory", "m", 0, "The amount of memory (in bytes) to use during garbage collection.")
+	garbageCollect.Flags().Int64VarP(&memoryBytes, "memory", "m", 0, "The amount of memory (in bytes) to use during garbage collection.")
```

Comment:
It would be nice if Cobra supported unit suffixed for byte measure .. somewhat like duration but alas... 
The doc might need to be changed to indicate this value being only in bytes. Right now it suggests suffixed and sort of implies value in MB .. just my thoughts :)

Reply:
Cobra doesn't support that, but no reason to let that stop us, `go-units` does.

Reply:
Not sure if my approval counts 😁  but you have my votes!  :shipit: